Tasks/Common/VstsAzureHelpers_/InitializeAzModuleFunctions.ps1

```diff
@@ -26,18 +26,40 @@ function Initialize-AzModule {
         Write-Verbose "Importing Az Modules."
         
         if ($featureFlags.retireAzureRM) {
-            $azAccountsModuleName = "Az.Accounts"
-            $azAccountsVersion = Import-SpecificAzModule -moduleName $azAccountsModuleName
-            Write-Verbose "'$azAccountsModuleName' is available with version $azAccountsVersion."
+            # Supress breaking changes messages
+            Set-Item -Path Env:\SuppressAzurePowerShellBreakingChangeWarnings -Value $true
+            $azureRMUninstalled = $false;
+
+            # If Az.Account already installed, remove RM modules
+            if (Get-Module -ListAvailable -Name Az.Accounts) {
+                Uninstall-AzureRMModules
+                $azureRMUninstalled = $true;
+            } else {
+                $azAccountsModuleName = "Az.Accounts"
+                $azAccountsVersion = Import-SpecificAzModule -moduleName $azAccountsModuleName
+                Write-Verbose "'$azAccountsModuleName' is available with version $azAccountsVersion."
+            }
 
+            # Update-AzConfig is a part of Az.Accounts
             if (Get-Command Update-AzConfig -ErrorAction SilentlyContinue) {
-                Write-Verbose "Supressing breaking changes warnings of '$($azAccountsModuleName)' module."
-                Update-AzConfig -DisplayBreakingChangeWarning $false -AppliesTo $azAccountsModuleName
+                Write-Verbose "Supressing breaking changes warnings of Az module."
+                Update-AzConfig -DisplayBreakingChangeWarning $false -CheckForUpgrade $false -AppliesTo Az
             } else {
                 Write-Verbose "Update-AzConfig cmdlet is not available."
             }
 
-            Uninstall-AzureRMModules -UseAzUninstall
+            if ($azureRMUninstalled -eq $false) {
+                Uninstall-AzureRMModules
+            }
+
+            # Enable-AzureRmAlias for azureRm compability
+            if (Get-Command Enable-AzureRmAlias -ErrorAction SilentlyContinue) {
+                Write-Verbose "Enable-AzureRmAlias for backward compability"
+                Write-Host "##[command]Enable-AzureRmAlias -Scope Process"
```

Comment:
Are we have some wrapper from sdk for this?

Reply:
Unfortunately, we have wrappers only for external programs and not for the pwsh methods